src/cli/commands/test/cache.rs

```diff
@@ -21,6 +31,52 @@ pub mod cache {
 		pub hash: String,
 	}
 
+	fn is_valid_cairo_contract(contract_path: &PathBuf) -> Result<(), CacheError> {
+		// check if contract_path have .cairo extension
+		let extension = contract_path
+			.extension()
```

Comment:
could you transform `InvalidContractExtension` into a transparent error?

Reply:
alright! then you may want to keep it that way:)
i'd advise to return the contract path in the `InvalidContractExtension` error rather than an empty string

Reply:
done!